### ISSUE TITLE
Would it be possible to use middle position without force_consent?

### ISSUE THREAD

**Issue by wzol** [orestbida/cookieconsent#95]
I was just wondering if there is a technical limitation behind it. I'd like to try it, maybe it would improve bounce rate.

**Comment by orestbida** [comment]
The only reason of why I don't want to vertically center the modal without `force_consent` enabled is due to legibility/clarity.

When `force_consent` is disabled, it means that there won't be a dark overlay behind the modal, and placing the modal centered on the screen (without a dark background) would make it hard to spot / distinguish from the rest of the page content.

**Comment by wzol** [comment]
Yes, I was thinking about this too and can't really decide. Probably it depends on the site and the visitor too. I can imagine a situation where it works, and I think I'd try it on my site and check the results. Maybe keeping the overlay but allowing scroll could work too? Can I just simply override the CSS or is it a deeper integration?

**Comment by orestbida** [comment]
yes, you can allow scrolling like so:

```css
html.force--consent.show--consent{
    overflow-y: auto!important;
}
```

you can also remove the overlay and restore the same behavior as if `force_consent` wasn't enabled (but allowing you this way to vertically center the modal):
```css
#cm-ov{
    display: none!important;
}

#cm{
    position: fixed!important;
}

#cc_div{
    position: unset!important;
}
```

**Comment by wzol** [comment]
Thank you, I'll try it.

**Comment by orestbida** [comment]
Due to an increasing number of requests, this option will be allowed to work without the need of having `force_consent` enabled. It will be "shipped" with the final 2.7 release.
